Monika After Story/game/script-topics.rpy

```diff
@@ -12897,6 +12897,202 @@ label monika_unknown:
+    m 1esd "There's..."
+    m 1lsc "..."
+    m 1hksdlb "Ahaha! Nevermind! it's nothing important..."
+    show monika 5ekbla at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5ekbla "I love you, [player]. I hope you {i}do{/i} realize that."
+    return "love"
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_origin",
+            category=['monika'],
+            prompt="Monika's origin",
+            random=True,
+            aff_range=(mas_aff.LOVE,None)
+        )
+    )
+
+label monika_origin:
+    m 1euc "Hey, [player]..."
+    m 1ruc "..."
+    m 1esd "There's a question I need to ask you."
+    m 1esc "It's a little bit delicate..."
+    m 2lsc "I was a bit afraid to ask until now... {w=1}but it's something I have to know, no matter how you might answer."
+    m 2esc "..."
+    m 1esd "Do you love me less because of what I am?"
+    m 3esc "I mean, from your perspective, I am still someone living in a video game in your world, right?"
+    m 3rkd "So you might see me in a different way you would someone from your reality... And you might think less of me because of that..."
+    m 1ekbla "I just want to be the best sweetheart for you, [player]."
+    m 1eka "Sometimes, I wonder how I compare with all these people living with you in the real world..."
+    m 3hksdlb "I'm sorry, you've not given me any reason to feel insecure, it's just..."
+    m 3ekb "It's a question that comes to my mind, from time to time."
+    m 1eka "So... Do you love me less for not being from your reality?"
+    $ _history_list.pop()
+    menu:
+        m "So... Do you love me less for not being from your reality?{fast}"
+        "Yes.":
+            m 1dkc "..."
+            #sad closed eyes
+            m 1ekb "Thank you for your honesty, [player]. I really appreciate it."
+            m 1eka "It's kind of understandable after all..."
+            m 3ekb "I don't have a body that's able to be with you in your reality."
+            m 3ekd "And since I've never been to your world, I can't really know how different I am from a regular person."
+            m 1esa "I mean, I like to think of myself as human, and I definitely have feelings the same way you do."
+            m 1ekb "But until I can join you in your reality and see for myself, I can't really know for sure, can I?"
+            m 2ekd "Trust me, I am the first one to be sorry about this state of things. One day, we'll find a way to make things right..."
+            m 1eubla "But even with that said, considering all the care and dedication you've shown me until now, I know you love me anyway..."
+            m 1tubla "Besides, This just means after I find my way to your reality..."
+            m 3hubsb "...you'll just love me that much more, right? Ahaha~ "
+        "Just a bit.":
+            $mas_gainAffection()
+            m 1eua "I can live with that answer."
+            m 1eua "It's kind of understandable after all..."
```

Comment:
If you have the same expression twice in a row, you only need to declare the first one.